lib/smart_answer/calculators/self_assessment_penalties.rb

```diff
@@ -30,8 +33,10 @@ def start_of_next_tax_year
         Date.new(2013, 4, 6)
       elsif tax_year == '2013-14'
         Date.new(2014, 4, 6)
-      else
+      elsif tax_year == '2014-15'
```

Comment:
What about adding a test to this method? https://github.com/alphagov/smart-answers/blob/07ff1d0751fc5841ec9baffc7eea82fc7d63d0d1/lib/smart_answer/calculators/self_assessment_penalties.rb#L26-L36